Dockerfile

```diff
@@ -1,73 +1,36 @@
-#syntax=docker/dockerfile:1.2
-# https://github.com/python-poetry/poetry/discussions/1879
-# `python-base` sets up all our shared environment variables
-FROM python:3.9-slim as python-base
+FROM python:3.9-slim as poetry
 
-# python
-ENV PYTHONUNBUFFERED=1 \
-	# prevents python creating .pyc files
-	PYTHONDONTWRITEBYTECODE=1 \
-	\
-	# pip
-	PIP_NO_CACHE_DIR=off \
-	PIP_DISABLE_PIP_VERSION_CHECK=on \
-	PIP_DEFAULT_TIMEOUT=100 \
-	\
-	# poetry
-	# https://python-poetry.org/docs/configuration/#using-environment-variables
-	POETRY_VERSION=1.1.5 \
-	# make poetry install to this location
-	POETRY_HOME="/opt/poetry" \
-	# make poetry create the virtual environment in the project's root
-	# it gets named `.venv`
-	POETRY_VIRTUALENVS_IN_PROJECT=true \
-	# do not ask any interactive question
-	POETRY_NO_INTERACTION=1 \
-	\
-	# paths
-	# this is where our requirements + virtual environment will live
-	PYSETUP_PATH="/opt/pysetup" \
-	VENV_PATH="/opt/pysetup/.venv"
+ARG BUILD_ESSENATIAL_VERSION='12.6*'
```

Comment:
Don't do this

Reply:
It's literally the recommended way to do it by Docker themselves so shut

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#general-guidelines-and-recommendations

Reply:
No

Reply:
Removed